### PR TITLE
add arpansagovau module for displaying UV data from Australian cities

### PR DESCRIPTION
**Pull request by jeffz**

### CODE REVIEW

File: modules/weatherservices/arpansagovau/settings.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+type Settings struct {
+	common *cfg.Common
+	city string;
+}
+
+func NewSettingsFromYAML(name string, ymlConfig *config.Config, globalConfig *config.Config) *Settings {
+	settings := Settings{
+		common: cfg.NewCommonSettingsFromModule(name, defaultTitle, ymlConfig, globalConfig),
+		city: ymlConfig.UString("locationid"),
```
Comment (senorprogrammer) [inline modules/weatherservices/arpansagovau/settings.go]: What happens if there is no city defined? Does it make sense to default it to something?

**Comment by senorprogrammer** [comment]
This looks great, it'll be included in the next release.

File: modules/weatherservices/arpansagovau/settings.go (hunk 1 above)
Comment (jeffz) [reply modules/weatherservices/arpansagovau/settings.go]: Good question.  If no city is defined, the widget displays "no data?" in red.   

As for a default, I don't feel too strongly about setting one.   

The accompanying PR for wtfdocs details a suggested default "Sydney" fwiw for the prospective user: https://github.com/wtfutil/wtfdocs/pull/53/commits/a79b9d87525abe30a48e3ba9fdeee9a03de8c9be - which I think is more useful to know than to have a default.

**Comment by senorprogrammer** [comment]
Thanks!

**Pull request merged**